scrapers_next/in/committees.py

```diff
@@ -0,0 +1,92 @@
+from spatula import URL, HtmlListPage, SkipItem, HtmlPage
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeDetail(HtmlPage):
+    example_source = URL(
+        "https://tlhgp53g3c.execute-api.us-east-2.amazonaws.com/beta/api/getMembers?committee_id=840b8a47-2a40-4e28-9e2a-68e6c66c9e45&session_lpid=session_2023"
+    )
+    name = None
+
+    def process_page(self):
+        com = self.input
+        if com:
+            for each_member in self.response.json()["members"]:
+                role = str(each_member["position"])
+                if role.lower().startswith("type"):
+                    role = role.replace("type_", "").replace("_", " ").title()
+                    if role.endswith("member") or role.endswith("chair"):
+                        role = role.replace("member", " Member").replace(
```

Comment:
If role includes "member", then role should be stored as just: "Member".

Currently, it's either getting stored as "Majority Member" or "Minority Member". We will already know whether the individuals are in the majority or the minority due to our other data on party affiliation. So just remove "majority" or "minority" when present in the string.